iggy-enrich-demo/iggy_enrich_demo.ipynb

```diff
@@ -0,0 +1,791 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "67acddf1-6d9b-4f21-8c51-a995e5c5291d",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "# Iggy Enrich Demo\n",
+    "\n",
+    "In this demo we will show some of the capabilities of Iggy Enrich data and show an example workflow for downstream task."
```

Comment:
I think we're calling it Iggy Place Data now (cc @lindsay-pettingill ?)

Reply:
May also want to include a brief tl;dr up here and/or a sentence saying what that downstream task is